docs/api/templates.md

```diff
@@ -155,29 +155,29 @@ curl -X GET http://coder-server:8080/api/v2/organizations/{organization}/templat
-| » build_time_stats                 | `codersdk.TemplateBuildTimeStats` | false    |              |                                            |
-| »» **additionalProperties**        | `codersdk.TransitionStats`        | false    |              |                                            |
-| »»» p50                            | integer                           | false    |              |                                            |
-| »»» p95                            | integer                           | false    |              |                                            |
-| » created_at                       | string                            | false    |              |                                            |
-| » created_by_id                    | string                            | false    |              |                                            |
-| » created_by_name                  | string                            | false    |              |                                            |
-| » default_ttl_ms                   | integer                           | false    |              |                                            |
-| » description                      | string                            | false    |              |                                            |
-| » display_name                     | string                            | false    |              |                                            |
-| » icon                             | string                            | false    |              |                                            |
-| » id                               | string                            | false    |              |                                            |
-| » name                             | string                            | false    |              |                                            |
-| » organization_id                  | string                            | false    |              |                                            |
-| » provisioner                      | string                            | false    |              |                                            |
-| » updated_at                       | string                            | false    |              |                                            |
-| » workspace_owner_count            | integer                           | false    |              |                                            |
+| Name                                 | Type                              | Required | Restrictions | Description                                |
+| ------------------------------------ | --------------------------------- | -------- | ------------ | ------------------------------------------ |
+| `*anonymous*`                        | array                             | false    |              |                                            |
```

Comment:
\* is ineffective inside code tags. But this also begs the question, what is anonymous?

Reply:
`Get templates by organization` returns an array of templates, which is considered to be an anonymous type. For workspaces, we're using `array of [codersdk.Workspace](#codersdkworkspace)`.

I'm not sure how we can improve this without breaking the API...

Reply:
Ah, that makes sense. I think `[array items]` was a good change. 👍🏻 Should we consider making it singular? `[array item]` / `[any property]`.

Reply:
👍 

Fixed